hw/opentitan/ot_spi_device.c

```diff
@@ -388,6 +397,17 @@ typedef enum {
     SPI_FLASH_ERROR, /* On error */
 } OtSpiFlashState;
 
+typedef enum {
+    SPI_TPM_IDLE, /* Wait for the fist byte of the header.*/
+    SPI_TPM_ADDR, /* Wait for the the follow 3 bytes of the header.*/
+    SPI_TPM_WAIT, /* Not ready for the host to read.*/
+    SPI_TPM_START_BYTE, /* Now read for the host to read.*/
```

Comment:
```suggestion
    SPI_TPM_ADDR, /* Wait for the following 3 bytes of the header.*/
    SPI_TPM_WAIT, /* Not ready for the host to read.*/
    SPI_TPM_START_BYTE, /* Now ready for the host to read.*/
```
Also as a formatting nit, put a space before the `*/`s? (many times in this file)